src/controls/inputs.js

```diff
@@ -492,21 +491,18 @@ Crafty.extend({
                         current.__c[comp] && current.isAt(pos.x, pos.y)) {
                         maxz = current._globalZ;
                         closest = current;
-                        layerPos = pos;
                     }
                 }
             }
         }
         
-        // If the pointer event isn't related to a specific layer, 
-        // find the Crafty position in the default coordinate set
-        if (!layerPos) {
-            layerPos = Crafty.domHelper.translate(x, y);
-        }
+        // Find the Crafty position in the default coordinate set,
+        // disregard the fact that the pointer event was related to a specific layer.
+        pos = Crafty.domHelper.translate(x, y);
 
         // Update the event coordinates and return the event target
-        e.realX = layerPos.x;
-        e.realY = layerPos.y;
```

Comment:
Great, thanks for taking a look at it! ❤️  Wish you a happy Easter weekend! See also comments in #1024.